setup.py

```diff
@@ -40,9 +40,6 @@
         'Operating System :: OS Independent',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
```

Comment:
Do we no longer want these?

Reply:
I believe more specific version classifiers are not needed if we support all active releases (currently [3.6 to 3.9](https://devguide.python.org/#status-of-python-branches)).

Reply:
We can add them back later if we feel they're necessary.